src/frontend/routing/Router.js

```diff
@@ -1,16 +1,26 @@
 import React from "react";
 import { Navigate, Route, Routes } from "react-router-dom";
-import { LandingPage, NotesPage, SignInPage, SignUpPage } from "../pages";
 
-const Router = () => {
+import { ArchivesPage, LabelsPage, LandingPage, NotesPage, SignInPage, SignUpPage, TrashPage } from "../pages";
+import { PrivateRoute } from "../components";
+import { useAuth } from "../context"
 
+const Router = () => {
+    const { auth: { status }, } = useAuth();
```

Comment:
Suggestion - You can destructure auth state in the Provider itself and pass on the it as value